modeldb/modeldb.py

```diff
@@ -1,84 +1,99 @@
-from . import config
+from .config import *
+from .data import Model
 import logging
 import multiprocessing
-import base64
 import os
+from pprint import pformat
+from .progressbar import ProgressBar
```

Comment:
Not a change that belongs in this pull request, but fyi there are Python libraries for progress bars, like [tqdm](https://tqdm.github.io/).